gapis/api/vulkan/extensions/khr_draw_indirect_count.api

```diff
@@ -0,0 +1,153 @@
+// Copyright (C) 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+///////////////
+// Constants //
+///////////////
+
+@extension("VK_KHR_draw_indirect_count") define VK_KHR_DRAW_INDIRECT_COUNT_SPEC_VERSION 1
+@extension("VK_KHR_draw_indirect_count") define VK_KHR_DRAW_INDIRECT_COUNT_EXTENSION_NAME "VK_KHR_draw_indirect_count"
+
+@internal class vkCmdDrawIndirectCountKHRArgs {
+  VkBuffer     Buffer
+  VkDeviceSize Offset
+  VkBuffer     CountBuffer
+  VkDeviceSize CountBufferOffset
+  u32          MaxDrawCount
+  u32          Stride
+}
+
+sub void dovkCmdDrawIndirectCountCOMMON!T(T draw) {
+  useRenderPass()
+  if draw.MaxDrawCount > 0 {
```

Comment:
nit: Everywhere this sub is called, it's already guarded with this check. Could make it a precondition of this sub - I think a comment would do.

Same for `dovkCmdDrawIndexedIndirectCountCOMMON`

Reply:
Done